sanityv3/schemas/documents/header/sharedHeaderFields.ts

```diff
@@ -76,7 +103,9 @@ const heroTitle = {
   of: [titleContentType],
   fieldset: 'header',
   hidden: ({ parent }: DocumentType) => {
-    return parent?.heroType !== HeroTypes.FIFTY_FIFTY
+    return (
+      parent?.heroType !== HeroTypes.FIFTY_FIFTY || (parent?.heroType === HeroTypes.FIFTY_FIFTY && parent.isBigTitle)
```

Comment:
logic should also be added to validation

Reply:
few more comments in here: https://github.com/equinor/energyvision/commit/081d9217100440b8005f2c3a92664135b107e5f1#comments